### DISCUSSION TITLE
三台如何实现互联

### DISCUSSION THREAD

**Discussion by clywm520** [opengnb/opengnb#33] [General]
请问一下,  Linux A、B、C服务器  ，如何仅运行gnb保持三者互相PING通
A有公网IP，可以访问互联网
B无公网IP，可以访问互联网
C无公网IP，不可以访问互联网，但与B存在内网互通。

请问一下,  Linux A、B、C服务器  ，如何仅运行gnb保持三者互相PING通

**Comment by gnbdev** [comment]
https://github.com/gnbdev/opengnb/blob/master/docs/gnb_user_manual_cn.md

**Comment by clywm520** [comment]
能给一个示例命令配置么，文档看半天就没有通过。
比如在A配置 或执行什么，B配置或执行什么 ，C配置或执行什么

**Reply by gnbdev** [reply]
If you can not understand the Chinese document，, We have English Document。
https://github.com/gnbdev/opengnb/blob/master/docs/gnb_user_manual_en.md
Or you should hire an engineer to help you.
